README.md

```diff
@@ -7616,6 +7616,12 @@ startap-script
 <summary>What the following commands does? `gcloud deployment-manager deployments create`</summary><br><b>
 </b></details>
 
+#### Google Kubernetes Engine (GKE)
+
+<details>
+<summary>What is GKE</summary><br><br>
```

Comment:
Missed that one

Reply:
Please check again because that was rectified long back